scipy/optimize/_linprog_util.py

```diff
@@ -33,13 +33,17 @@
     b_eq : 1D array, optional
         The equality constraint vector. Each element of ``A_eq @ x`` must equal
         the corresponding element of ``b_eq``.
-    bounds : sequence, optional
-        A sequence of ``(min, max)`` pairs for each element in ``x``, defining
-        the minimum and maximum values of that decision variable. Use ``None`` to
-        indicate that there is no bound. By default, bounds are ``(0, None)``
-        (all decision variables are non-negative).
-        If a single tuple ``(min, max)`` is provided, then ``min`` and
-        ``max`` will serve as bounds for all decision variables.
+    bounds : various valid formats, optional
+        The bounds of ``x``, as ``min`` and ``max`` pairs.
+        If bounds are specified for all N variables separately, valid formats are:
+        * a 2D array (2 x N or N x 2);
+        * a sequence of N sequences, each with 2 values.
```

Comment:
We can still accept this, because `np.array(bounds)` will turn this into an `N x 2` array.

Reply:
You mean the sequence of pairs?

Reply:
Yes